extensions/android-paging3/build.gradle

```diff
@@ -1,9 +1,15 @@
 plugins {
-  alias(deps.plugins.kotlin.jvm)
+  alias(deps.plugins.android.library)
+  alias(deps.plugins.kotlin.android)
   alias(deps.plugins.publish)
   alias(deps.plugins.dokka)
 }
 
+android {
+  namespace 'app.cash.sqldelight.paging3'
+  compileSdkVersion 23
+}
+
```

Comment:
I don't want to make this an Android Library, but instead keep in a JVM library. I'm not too sure how do this though, as I need Android dependencies just for the tests. Any ideas?

Reply:
Put the tests in another module. E.g., https://github.com/square/retrofit/tree/master/retrofit/android-test

Reply:
Thanks! Reverted back to a JVM artifact in https://github.com/cashapp/sqldelight/pull/3396/commits/71501d207a642b4d8f84882d10431a120cb6cc7f